templates/pretext-cli.yml

```diff
@@ -24,8 +24,21 @@ jobs:
             - name: install deps
               run: pip install -r requirements.txt
 
+            - name: install local ptx files
+              run: pretext --version
+
             - name: build deploy targets
-              run: pretext build --deploys
+              run: |
+                version="$(pretext --version)"
+                major="$(echo $version | cut -d '.' -f 1)"
+                minor="$(echo $version | cut -d '.' -f 2)"
+                if [ "$major" -ge 2 -a "$minor" -ge 5 ]; then
+                    echo "PreTeXt version is 2.5 or greater; using new build command"
+                    pretext build --deploys
+                else
+                    echo "PreTeXt version is less than 2.5, using old build command"
+                    pretext build
+                fi
```

Comment:
Is there a clever way we can configure `pretext-cli.yml` to know the version of PreTeXt that generated it? I'd just as soon reject any attempt to use CI/CD with a mismatched PreTeXt version and GitHub action.

Reply:
If a user generates the `pretext-cli.yml` with 2.6 but decides to keep 2.4 in `requirements.txt` then it would still fail.